ucp/useraccounts_module.php

```diff
@@ -203,10 +164,10 @@ function main($id, $mode)
 			case 'manage_keys':
 			{
 				// Set up vars for mode.
-				$start = $this->request->variable('start', 0); // Used for pagination.
-				$markedkeys = $this->request->variable('markedkeys', array(0)); // items ids.
-				$revokemark = $this->request->is_set_post('markedkeys') && $action == 'rev_marked' ? true : false;
-				$revokeall = $action == 'rev_all' ? true : false;
+				$start = (int)$this->request->variable('start', 0); // Used for pagination.
+				$markedkeys = (array)$this->request->variable('markedkeys', array(0)); // items ids.
```

Comment:
`array(0)` is an array containing the number `0`.  Shouldn't this be an empty array (`array()`)?